src/js/providers/tracks-mixin.ts

```diff
@@ -985,4 +990,16 @@ function cuesMatch(cue1: TrackCue, cue2: TrackCue): boolean {
         JSON.stringify(cue1.value) === JSON.stringify(cue2.value);
 }
 
+function hideIfOldTrackSafari(tracks: TextTrackList, track: TextTrackLike, trackId: string): boolean {
+    if (Browser.safari) {
+        if (track.parsed && !tracks[trackId]) {
```

Comment:
What assigns `parsed` to a track? Can you add a comment here?

Reply:
Added comment